ext/openssl/ossl_asn1.c

```diff
@@ -654,18 +654,18 @@ ossl_asn1_class2sym(int tc)
  *   asn1_int = OpenSSL::ASN1Data.new(42, 2, :UNIVERSAL) # => Same as OpenSSL::ASN1::Integer.new(42)
  *   tagged_int = OpenSSL::ASN1Data.new(42, 0, :CONTEXT_SPECIFIC) # implicitly 0-tagged INTEGER
  */
-static VALUE
-ossl_asn1data_initialize(VALUE self, VALUE value, VALUE tag, VALUE tag_class)
-{
-    if(!SYMBOL_P(tag_class))
-	ossl_raise(eASN1Error, "invalid tag class");
-    ossl_asn1_set_tag(self, tag);
-    ossl_asn1_set_value(self, value);
-    ossl_asn1_set_tag_class(self, tag_class);
-    ossl_asn1_set_indefinite_length(self, Qfalse);
-
-    return self;
-}
+// static VALUE
+// ossl_asn1data_initialize(VALUE self, VALUE value, VALUE tag, VALUE tag_class)
+// {
+//     if(!SYMBOL_P(tag_class))
+// 	ossl_raise(eASN1Error, "invalid tag class");
+//     ossl_asn1_set_tag(self, tag);
+//     ossl_asn1_set_value(self, value);
+//     ossl_asn1_set_tag_class(self, tag_class);
+//     ossl_asn1_set_indefinite_length(self, Qfalse);
+
+//     return self;
+// }
```

Comment:
Why did you comment out by `//` rather than removing the parts?

Reply:
This is an early draft, just wanted to get an opinion about the approach before moving forward (still some more work to be done).

Reply:
done